libcxx/test/std/numerics/rand/rand.dist/rand.dist.bern/rand.dist.bern.bin/eval.pass.cpp

```diff
@@ -151,8 +151,8 @@ void test3() {
     double x_kurtosis = (1-6*d.p()*(1-d.p())) / x_var;
     assert(std::abs((mean - x_mean) / x_mean) < 0.01);
     assert(std::abs((var - x_var) / x_var) < 0.01);
-    assert(std::abs((skew - x_skew) / x_skew) < 0.03);
-    assert(std::abs((kurtosis - x_kurtosis) / x_kurtosis) < 0.3);
```

Comment:
Why did you change this test?

Reply:
Oops, I was debating changing it to an absolute test because the expected kurtosis is close to zero (-0.017). 2.0 should be the relative bound.

For reference, if you approximate the binomial distribution as normal, you would expect the sample kurtosis to have a standard deviation of $\sqrt{24/N} \approx 0.0155$. That gives a relative 90% confidence interval of $\sqrt{2}\mathrm{erf}^{-1}(0.9)\cdot 0.0155 / 0.017 \approx 1.5$, so I don't think the empirically observed value of 2.0 is unreasonable.

Reply:
Can you add something along these lines in the commit message. That give future readers more information why these changes are valid.

Reply:
@MattStephanson can you update the commit message? Then the patch is good to land. Thanks.

Reply:
@mordante I did update it the last time I force-pushed [788fc37](https://github.com/llvm/llvm-project/commit/788fc376ccd4084d4e7ba916da3ddae5a7944c46). If it's still missing something, could you please be more specific? Thanks!

Reply:
Oh, okay. I think I have it now?

Reply:
Yes great! Thanks. I'll land the patch.